kube-burner-workload/env

```diff
@@ -21,13 +21,18 @@ KUBECONFIG=
 
 # PLATFORM is one of the folders under network-policy workload
 PLATFORM=kind-metrics
-# CONVERGENCE_TRACKER
+# Convergence tracker settings
 CONVERGENCE_TRACKER=
+# CONVERGENCE_PERIOD and CONVERGENCE_TIMEOUT are convergence tracker parameters.
+# CONVERGENCE_PERIOD specifies for how long the system should be stable to be considered converged and
+# CONVERGENCE_TIMEOUT is a timer specifying the hard deadline for policy convergence. An expired timer
+# indicates a test failure. CONVERGENCE_TIMEOUT should be a larger than CONVERGENCE_PERIOD.
```

Comment:
so as it is used now,
CONVERGENCE_TIMEOUT = timeout for a system to converge + CONVERGENCE_PERIOD
may be it will be easier if we change convergence tracker to do `timeout = convergence_timeout + convergence_period`? Then we don't need any extra requirements like `CONVERGENCE_TIMEOUT should be a larger than CONVERGENCE_PERIOD.`

Reply:
should we remove `CONVERGENCE_TIMEOUT should be a larger than CONVERGENCE_PERIOD.`? and may be say 
`A test failure will be reported by convergence tracker in CONVERGENCE_TIMEOUT + CONVERGENCE_PERIOD` seconds?